inject-java/src/main/java/io/micronaut/annotation/processing/visitor/JavaMethodElement.java

```diff
@@ -75,6 +75,15 @@ public boolean isDefault() {
         return executableElement.isDefault();
     }
 
+    @Override
+    public boolean overrides(MethodElement methodElement) {
+        return visitorContext.getElements().overrides(
```

Comment:
surround with `instanceof JavaMethodElement` to avoid case where `null` is passed and or a non `JavaMethodElement`